assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -161,14 +199,23 @@ function Carousel() {
 
   const isAtBeginningOfList = 0 === scrollPercentage;
   const isAtEndOfList = 1 === scrollPercentage;
+  const scrollByPx = carouselSize.width;
+
+  const Item = isCompact ? CompactIndicator : DraggablePage;
+  const itemMarginRight = isCompact
+    ? CAROUSEL_COMPACT_RIGHT_MARGIN
+    : CAROUSEL_THUMB_RIGHT_MARGIN;
+  const [pageThumbWidth, pageThumbHeight] = calculatePageThumbSize(
+    carouselSize
+  );
 
   return (
     <>
-      <Wrapper>
-        <Area area="left-navigation">
+      <Wrapper ref={ref}>
```

Comment:
This `ref` doesn't seem to be used anywhere?

Reply:
Yes, I looked at the changes before push and realized that I can reuse existing `useResizeEffect on listRef` and moved the stuff there.

Reply:
Cool, let's remove it then 👍